docs/Unicode.md

```diff
@@ -54,6 +54,15 @@ From the error in the image above, Thai characters can be fixed using fonts from
 
 ![thai-font-working](https://raw.githubusercontent.com/PyFPDF/fpdf2/master/tutorial/thai-accent-working.png)
 
+*Scripts with automatic ligatures and contextual forms (eg. Indic scripts and Abjads like Devanagari, Hebrew, Arabic) currently have limited support. For the Arabic script there is a temporary solution (using two additional libraries) that works for most languages; only a few (rare) characters aren't supported. Using it on other scripts(eg. when the input is unknown or mixed scripts) does not affect them:
```

Comment:
This section is already a bit long,
would you mind starting this section with a `### Arabic script workaround` header please?

Reply:
done. Should I add it to `Text.md` too or is it fine like that?

Reply:
It's fine like that 😊